.gitignore

```diff
@@ -12,3 +12,6 @@ __pycache__
 .vercel
 .env
 !manifest.json
+bin
```

Comment:
These are the folders created when you initialize a virtual environment on a Mac or Linux computer for local development.

Reply:
Ok make sense